doctors_surgery.html

```diff
@@ -84,51 +84,49 @@ <h2>Need advice or have some feedback for us?</h2>
 
         <p>Send us a message by filling out the form below and one of our Receptionists will be in touch:</p>
 
-        <form action="" method="get">
+        <form>
 
-            <div>
+            <fieldset>
```

Comment:
Would a fieldset work here? It looks a bit funny without any styling, but I wondered if it might be a bit more descriptive than a div?